src/stories/Breadcrumbs.story.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import Breadcrumbs from '../lib/Breadcrumbs';
+import { Row, StoryItem } from './utils.js';
+
+const stories = storiesOf('Core', module);
+
+stories
+  .addDecorator(withKnobs)
+  .addWithInfo('Breadcrumbs', '', () => (
+    <div>
+      <Row title="Theme">
+        <StoryItem title="Default">
+          <Breadcrumbs
+            links={[
+              {"href" : "https://10clouds.com", "icon": "fa fa-home"},
+              {"href" : "https://10clouds.com", "text" : "Second page"},
+              {"href" : "https://10clouds.com", "text" : "Third page"}
+              ]}
+          />
+        </StoryItem>
+        <StoryItem title="With shadow">
+          <Breadcrumbs
+            shadow={true}
+            links={[
+              {"href" : "https://10clouds.com", "icon": "fa fa-home"},
+              {"href" : "https://10clouds.com", "text" : "Second page"},
+              {"href" : "https://10clouds.com", "text" : "Third page"}
+              ]}
+          />
+        </StoryItem>
+      </Row>
+
+      <Row title="Crumb style">
+        <StoryItem title="As icon">
+          <Breadcrumbs
+            links={[
+              {"href" : "https://10clouds.com", "icon": "fa fa-home"},
+              {"href" : "https://10clouds.com", "text" : "Second page"}
+              ]}
+          />
+          <Breadcrumbs
+            shadow={true}
```

Comment:
You can also write 
` <Breadcrumbs shadow />`

Reply:
OK